beers/models.py

```diff
@@ -38,18 +40,45 @@ class BeerStyle(models.Model):
     category = models.ForeignKey(BeerStyleCategory, on_delete='CASCADE')
     tags = models.ManyToManyField(BeerStyleTag, blank=True)
 
-    ibu_low = models.PositiveSmallIntegerField(default=0)
-    ibu_high = models.PositiveSmallIntegerField(default=0)
-    srm_low = models.PositiveSmallIntegerField(default=0)
-    srm_high = models.PositiveSmallIntegerField(default=0)
+    ibu_low = models.PositiveSmallIntegerField(
+        'Minimum bitterness (International Bitterness Units)', default=0,
+    )
+    ibu_high = models.PositiveSmallIntegerField(
+        'Maximum bitterness (International Bitterness Units)', default=0,
+    )
+    srm_low = models.PositiveSmallIntegerField(
```

Comment:
SRM is an integer here and Decimal in Beer.

Reply:
If the brewers use the calculated SRM value, then it'll be decimal. If they're doing it against reference samples in a lab, then it'll be an integer.

Have you seen a data source that actually gives us SRM?

Reply:
Uhhhh. No. I have not.

Reply:
I think for now we can leave it alone and then migrate if needed later.